orchagent/orch.h

```diff
@@ -149,6 +150,9 @@ class Orch
 
     vector<Selectable*> getSelectables();
 
+    // add the existing table data (left by warm reboot) to the consumer todo task list.
+    bool addExistingData(Table *table);
```

Comment:
There are quite a few places to use addExistingData() where only table name is available.   It should be better to use table name directly.